default

```diff
@@ -0,0 +1,22 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	root /var/www/html/wordpress;
+
+	index index.php index.html index.htm;
+
+	#server_name localhost;
+	server_name XXXXXX;
```

Comment:
do you actually need server_name here?

Reply:
Yes, it's used by nginx on every wordpress instance. I put XXXXX just as filler, they are replaced with each wordpress instance's ip direction (wordpress.yaml playbook, line 62)